portfolio/src/main/webapp/write-message.html

```diff
@@ -6,20 +6,24 @@
     <link rel="stylesheet" href="style.css" />
     <script src="script.js"></script>
   </head>
-  <body onload="addMessage()" id="content">
-    <h1>Write A Message</h1>
-    <h2>Prior Messages</h2>
-    <ul id="past-messages"></ul>
+  <body id="content" onload="addMessage()">
     <form action="/data" method="post" id="message-form">
+      <h1>Write A Message</h1>
+      <h2>Prior Messages</h2>
+      <ul id="past-messages"></ul>
+      <h2>Newest Posted Message</h2>
+      <ul id="current-message"></ul>
+
       <h2>Enter Name</h2>
       <textarea name="username" placeholder="Type Username"></textarea>
 
       <h2>Enter Message Here</h2>
       <textarea name="message-data" placeholder="Write a message about the webpage"></textarea>
+      <input type="number" id="comment-field" name="comment-limit" min="1" max="10" value="5" />
```

Comment:
Should you add an empty line in between the textarea and the input?

Reply:
Is that a common practice?

Reply:
Sorry, I meant a line break `<br />`

Reply:
Yes, I meant to put one there thank you for pointing that out.